questions/106-medium-trimleft/template.ts

```diff
@@ -1 +1,3 @@
-export type TrimLeft<S extends string> = any
+type TrimLeft<T extends string> = T extends ` ${infer Rest}` ? TrimLeft<Rest> : T;
```

Comment:
would this work with '\n' or '\t' ?

Reply:
In my solution \n or \t wouldn't work, because this **_only trims spaces_** if I want to work with newlines and tabs I think I can do it this way 
T extends `\n${infer Rest}` ? TrimLeft<Rest> : //NEW LINE 
T extends `\t${infer Rest}` ? TrimLeft<Rest> : //TAB